svg/svgGenerator.ts

```diff
@@ -0,0 +1,25 @@
+import {promises as Fs} from "fs";
+
+export async function generate(title: string): Promise<string> {
+    let svgAsString = await read();
+    return replaceInSvg(svgAsString, title);
+}
+
+function replaceInSvg(svg:string, title: string): string {
+    return svg.replace("TITLE_TEMPLATE", title);
+}
+
```

Comment:
Alternativt kunne man bruge template-literals, så man slipper for en ekstra fil og streng-manipulation. Den burde også være nem at udvide :grinning: 
```
function template(title) {
  return `
    <svg width="100" height="200" xmlns="http://www.w3.org/2000/svg" >
      <rect width="100" height="200" fill="yellow"/>
      <text id="title" x="50%" y="50%" dominant-baseline="middle" text-anchor="middle" fill="blue">${title}</text>
    </svg>
  `
}
```

Reply:
den er jeg også umiddelbart mere til